### ISSUE TITLE
NURBS hexagonal segment union solid fails

### ISSUE THREAD

**Issue by Symbian9** [solvespace/solvespace#1205]
### System information

- **SolveSpace version:** `3.0~`https://github.com/solvespace/solvespace/commit/b399d9a1ec50086cdfc827a43d661ef79052e175
- **Operating system:** Debian 10 buster

### Expected behavior

There should not be fails in NURBS segment union solid

### Actual behavior

NURBS solid fails.

### Additional information

- [slvs3x-hex-hole_asm.zip](https://github.com/solvespace/solvespace/files/7974122/slvs3x-hex-hole_asm.zip)

<details><summary>Sreenshot:</summary>

![Screenshot_2022-01-31_22-33-13](https://user-images.githubusercontent.com/13188580/151869517-a947e20a-d909-4a90-b3b5-d90f988b2e5a.png)

</details>

NOTE: Instead of https://github.com/solvespace/solvespace/issues/1204 (where same metric segment solid used), here I used corrected cut solid as a segment for hexagonal hole.

**Comment by simrim1** [comment]
Click the group g004-rotate to show its properties. Then check the property "force NURBS surfaces to triangle mesh". It will fix the error.

![NURBSSurfaces](https://user-images.githubusercontent.com/13056546/152033166-f0eeb861-e846-49a2-8452-d2f2356a3bd3.png)
.

**Comment by Symbian9** [comment]
> Then check the property "force NURBS surfaces to triangle mesh". It will fix the error.

You forced triangle mesh. That is out of scop of this *NURBS solid bug*.

**Comment by simrim1** [comment]
@Symbian9, That is correct. I am new here and sorry if it was out of scope of your issue. I am deleting my reply. Sorry again.

**Comment by phkahler** [comment]
@simrim1 dont worry about it :-)  he is correct though. When NURBS surfaces fail, one "solution" is to force to triangle mesh but that is only viable in some cases. You won't be able to export a STEP file for example. Even though it masks a NURBS issue, there is still an issue that we would like to fix.
